stories/topics/ref-aap-gcp-install-permissions.adoc

```diff
@@ -0,0 +1,88 @@
+[id="ref-aap-gcp-install-permissions"]
```

Comment:
Don't forget you'll have to add this file to the assembly.

assembly-aap-gcp-install.adoc

Add it after this line

include::topics/ref-aap-gcp-install-prerequisites.adoc[leveloffset=+1]
in the following form
include::topics/ref-aap-gcp-install-permissions.adoc[leveloffset=+1]

(make it +2 if you want it to be a subsection of the Prerequisites.)